includes/admin/assets.php

```
@@ -56,6 +56,13 @@ public function load()
 		 * Register Admin Scripts *
 		 * ====================== */
 
+		// Use file mtimes to avoid stale admin asset caches during development/releases.
+		$admin_js_ver = SIMPLE_CALENDAR_VERSION;
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$admin_js_ver'. (undefined)

(UnusedLocalVariable)